tests/src/main.rs

```diff
@@ -276,9 +278,10 @@ async fn main() -> Result<()> {
             _ => bail!("Expected a job assignment"),
         };
         let rebuild = Rebuild {
+            status: BuildStatus::Good,
```

Comment:
Probably more editor shenanigans, but we don't want this line (and others like it) getting moved around, as it can litter the history :)

Reply:
+1